buildSrc/src/main/kotlin/ort-server-kotlin-conventions.gradle.kts

```diff
@@ -45,3 +47,7 @@ detekt {
 
     basePath = rootDir.path
 }
+
+tasks.register("detektAll") {
+    dependsOn(tasks.withType<Detekt>())
+}
```

Comment:
Doesn't this need a description so that it appears in `./gradlew tasks` without `--all`?